app/tests/unit/containers/User/UserProfileDropdown.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {UserProfileDropdownComponent} from 'containers/User/UserProfileDropdown';
+
+// https://github.com/facebook/relay/issues/2394#issuecomment-379590645
+const mockRefType: any = null;
+const user = {
+  firstName: 'Hamza',
+  lastName: 'Javed',
+  emailAddress: 'hamza@button.is',
+  ' $refType': mockRefType
```

Comment:
```suggestion
  ' $refType': 'UserProfileDropdown_user'
```

this should just be the name of the relay fragment/query